policies/committer-policy.md

```diff
@@ -0,0 +1,102 @@
+
+# Policy for OpenSSL Committers
+
+## Who is a committer?
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.  Committers review and commit their own patches
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by the [OMC] typically on the recommendation
+of the [OTC] (see the [OpenSSL Bylaws]).
+
+We welcome contributors who become domain experts in some part of
+the library (for example, low-level crypto) as well as generalists
+who contribute to all areas of the codebase.  All committers share
+the responsibility for the overall health of the project: aside from
+contributing quality features, committers are team players who fix bugs,
+address open issues, review community contributions, and improve tests
+and documentation.  Committers are also shepherds of the OpenSSL community
+and its code of conduct.
+
+To become a committer, start by contributing code.  Read our coding style,
+and get to know our build and test system.  Then, use the Github issue
+tracker, and our mailing lists find impactful ideas to work on.
+
+## How to maintain committer status?
+
+To maintain committer status, you must stay active in the project.  Refer
+to the OpenSSL Bylaws for details.
+
+In the unlikely and unfortunate event that your actions conflict with
+the project objectives or are otherwise disruptive, committer status
+may also be revoked by the OMC.
+
+## Approvals and code reviews
+
+All submissions must be reviewed and approved by at least two committers,
+one of whom must also be an OTC member.  Neither of the reviewers can
+be the author of the submission.
+
+The sole exception to this is for the release procedure commits where
```

Comment:
There are different commits involved in the release. There are those automatically generated ones which come from the scripts. Also we often end up doing things like updating CHANGES/NEWS etc. We will have to get better at doing those ones earlier. I do worry that there will inevitably things that crop up during the release that we have to fix at the last moment - but I guess we'll have to see how that works out in practice. Should this statement in the policy be more explicit about referring to the automatically generated release procedure commits?

Reply:
I omitted the automatically generated part on purpose.  I think it will cause more pain than it is worth.
I've no problem with adding it, it's someone else's pain :wink: 

We could make an attempt to track the manual changes to see how impactful it would be and add that later.

Reply:
I think we should be explicit about the automatically generated stuff. The alternative is that we allow a broader range of changes while doing the release procedure - but then I think it gets a lot more difficult to nail down exactly what is and is not allowed. The auto generated stuff is at least easy to define.

Reply:
I've changed this to the automated changes plus NEWS and CHANGES edits.